### PR TITLE
Add cluster-svcat-controller-manager-operator

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

**Comment by jwforres** [comment]
/hold

associated repo this will test doesn't have the right content in it yet

cc @jboyd01

File: ci-operator/config/openshift/cluster-svcat-controller-manager-operator/openshift-cluster-svcat-controller-manager-operator-master.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+- as: e2e-aws-operator
+  commands: make test-e2e
+  openshift_installer_src:
+    cluster_profile: aws
+
+resources:
+  '*':
+    limits:
+      cpu: '6'
```
Comment (petr-muller) [inline ci-operator/config/openshift/cluster-svcat-controller-manager-operator/openshift-cluster-svcat-controller-manager-operator-master.yaml]: I know I approved the previous one which had the same but CI jobs should not have CPU limits now unless it's 100% intentional. See https://github.com/openshift/release/pull/2821#issuecomment-462393233

File: ci-operator/config/openshift/cluster-svcat-controller-manager-operator/OWNERS [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# DO NOT EDIT; this file is auto-generated using tools/populate-owners.
+# from https://github.com/openshift/cluster-svcat-controller-manager-operator/blob/a5c077c0a48ded6c0247f51c9a6079ddf6aead1d/OWNERS
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- jboyd01
+reviewers:
+- jboyd01
```
Comment (petr-muller) [inline ci-operator/config/openshift/cluster-svcat-controller-manager-operator/OWNERS]: Who is going to `/lgtm` jboyd01's PRs? :)

**Comment by jwforres** [comment]
updated based on @petr-muller 's feedback, fixed up the svcat apiserver operator stuff as well

**Comment by jwforres** [comment]
hold on, think i missed some cases of limits

**Comment by petr-muller** [comment]
We've updated prowgen today, make sure you're using the latest image to generate the jobs

**Comment by jwforres** [comment]
ahhhh ok thats what's going on, thanks

**Comment by jwforres** [comment]
ok i thinks this just needs an lgtm now @petr-muller  ?

**Pull request merged**